### DISCUSSION TITLE
Importing Json file from another district

### DISCUSSION THREAD

**Discussion by webdist17** [code4recovery/12-step-meeting-list#1167] [Q&A] [answered]
I am webmaster for our district website which I installed TSML. I want import from a data source at our Intergroup website which  is already installed and connected to the meeting app. The import setting is looking for a Json file and I don't see a permanant one on the Intergroup website.  The intergroup website does have some Json files in it's cache folder with the name of 
tsml-cache-82b71d8423.json and these can successfully be connected to our site but these appear to be temporary files created when Intergroup updates the meeting app data.
Is there a permanent json file to be imported? Where is it? 

Sorry if i am not making sense, I am new to this and only trying to help my district.

thanks,

pete

**Comment by joshreisner** [comment]
hi pete, this sounds like a 12 Step Meeting List (WordPress plugin) question - going to move it over there, so other people can find it.

this is the TSML UI frontend project

**Reply by joshreisner** [reply, marked as answer]
ok it's moved now. there are some instructions here: https://meetingguide.helpdocs.io/article/tjn0po3zco-providing-meeting-listings-to-another-entity on how to provide meeting data to another entity. someone with access to the intergroup site should follow these instructions to provide a URL and then you should be able to import that URL to the district site

**Reply by webdist17** [reply]
Thanks. I finally read that document correctly and it works. Have a great 24!
